swagger.yaml

```diff
@@ -4121,6 +4121,12 @@ definitions:
       fresh:
         type: boolean
         description: Has the Topic not been answered to in a configurable timeframe, after being created? This status only is true when no answer was given yet.
+      edited_by:
+        type: integer
+        description: id of a user that last updated the topic
```

Comment:
... this is really what we added?
we usually shouldn't expose the id of anything in the APIv2, 1 because of some vague security concerns, 2 because you can't request a user with an id, this data is virtually useless

Reply:
Yes, it's defined here https://github.com/brandslisten/bl/pull/4031/files#diff-5c6897ff73d1ebdbffbba06ccf708e59db2b6a21f146be7d3c8e459452521c41R9

I think its worth to make a separate dev meeting to discuss such aspects, coz Im not aware of it

Reply:
yeah, probably
and for core it makes sense to return the id, there we have access to everything, but in the APIv2 we don't and we don't want to give it to the consumer

would you schedule a meeting or should I?